src/schema/search.ts

```diff
@@ -388,6 +410,37 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         .limit(getSearchLimit({ limit }));
       const hits = await searchQuery.getRawMany();
 
+      return {
+        query,
+        hits,
+      };
+    },
+    searchUserSuggestions: async (
+      source,
+      { query, limit }: SearchSuggestionArgs,
+      ctx,
+    ): Promise<GQLSearchSuggestionsResults> => {
+      const searchQuery = ctx.con
+        .createQueryBuilder()
+        .select(`id, name as title, username as subtitle, image`)
+        .from('user', 'u')
+        .where('u.infoConfirmed = TRUE')
+        .andWhere("u.id != '404'")
+        .andWhere(
+          new Brackets((qb) => {
+            return qb
+              .where(`name ILIKE :query`, {
+                query: `%${query}%`,
+              })
+              .orWhere(`username ILIKE :query`, {
+                query: `%${query}%`,
+              });
+          }),
+        )
+        .andWhere(whereVordrFilter('u'))
```

Comment:
Yup, looks good. Should return `COALESCE((u.flags ->> 'vordr')::boolean, false) = false`